sss/sss_table.py

```diff
@@ -155,17 +305,13 @@ def read_file(file):
         print('This file cannot be read' + ' ' + file.split('/')[-1])
     return df, file
 
-
-def pre_check(df):
-    print(df.dtypes)
-
-
 def check_extra_columns(df):
     """
     This function takes in a pandas dataframe and adds a boolean column(s)
```

Comment:
I think it actually always adds three boolean columns so you can be more specific here. And I think you can drop some of the beginning to make space for more specificity. You don't need to say "This function" because we know it's a function docstring. Maybe something like: "Adds three boolean columns to the sss dataframe to indicate cost breakdowns in other tables".